packages/core/pyproject.toml

```diff
@@ -43,7 +43,7 @@ pillow = "^8.4.0"
 [tool.poetry.dev-dependencies]
 black = {version = "^21.5b0", allow-prereleases = true}
 flake8 = "^3.7.9"
-pylint = "^2.4.4"
+pylint = "^2.4.4, <2.13"
```

Comment:
If you don't want to have "fun" with releasing all our packages, maybe pinning this for now into dialogs and main packages should be enough. (but I guess you know best and it's even better to fix this problem everywhere else it happens)

Good luck!

Reply:
well, I think this is just a version pin and shouldn't be released; it's just for the developer experience so that whoever works on those projects in the future won't have the pipeline failing. Is there a reason why a release would be necessary for that?

Reply:
May bad, since it's dev only then there's no reason to do that as this gets used with `poetry install`s only. (but never published afaik)

Reply:
May bad, since it's dev only then there's no reason to do that as this gets used with `poetry install`s only. (but never published afaik)